### PR TITLE
Factoring out ResourceService::put a bit

### PR DESCRIPTION
**Pull request by ajs6f**

### CODE REVIEW

**Comment by acoburn** [comment]
You’d need to add a dependency on trellis-vocabulary for this to work. Or, you could just remove the two-argument put method.

**Comment by ajs6f** [comment]
Your call. I left the two-arg in to be less of a jarring change, but it's all pre-1.0 anyway...

**Comment by acoburn** [comment]
I’d suggest just removing it.

File: src/main/java/org/trellisldp/api/ResourceService.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 import static org.trellisldp.api.RDFUtils.TRELLIS_BNODE_PREFIX;
 import static org.trellisldp.api.RDFUtils.TRELLIS_PREFIX;
 import static org.trellisldp.api.RDFUtils.getInstance;
+import static org.trellisldp.vocabulary.Trellis.PreferServerManaged;
```
Comment (acoburn) [inline src/main/java/org/trellisldp/api/ResourceService.java]: You can remove this line

File: src/main/java/org/trellisldp/api/ResourceService.java [hunk 2]
```diff
@@ -30,6 +31,7 @@
 import org.apache.commons.rdf.api.Quad;
 import org.apache.commons.rdf.api.RDFTerm;
 import org.apache.commons.rdf.api.Triple;
+import org.trellisldp.vocabulary.RDF;
```
Comment (acoburn) [inline src/main/java/org/trellisldp/api/ResourceService.java]: You can also remove this line

**Comment by ajs6f** [comment]
Oops-- sorry. I even have Eclipse set to warn on that, just didn't notice.

**Comment by ajs6f** [comment]
Urg, I'll get it right someday! For the future we might want to go to returning `Future<Boolean>`.

**Comment by acoburn** [comment]
and changing the return type to a `Future<Boolean>` would be great.

**Pull request merged**